docs/user/openstack/README.md

```diff
@@ -105,63 +102,106 @@ openstack network list --long -c ID -c Name -c "Router Type"
 +--------------------------------------+----------------+-------------+
 ```
 
-**NOTE:** If the `neutron` `trunk` service plugin is enabled, trunk port will be created by default. for more information, please refer to [neutron trunk port](https://wiki.openstack.org/wiki/Neutron/TrunkPort).
+**NOTE:** If the `neutron` `trunk` service plug-in is enabled, trunk port will be created by default. For more information, please refer to [neutron trunk port](https://wiki.openstack.org/wiki/Neutron/TrunkPort).
 
-### Isolated Development
+## Isolated Development Environment
 
-If you would like to set up an isolated development environment, you may use a
-bare metal host running CentOS 7.  The following repository includes some
-instructions and scripts to help with creating a single-node OpenStack
-development environment for running the installer.  Please refer to the
-documentation in that repository for further details.
+If you would like to set up an isolated development environment, you may use a bare metal host running CentOS 7. The following repository includes some instructions and scripts to help with creating a single-node OpenStack development environment for running the installer. Please refer to [this documentation](ocp-doit) for further details.
 
-* https://github.com/shiftstack-dev-tools/ocp-doit
+[ocp-doit]: https://github.com/shiftstack-dev-tools/ocp-doit
 
-## API Access
+## Initial Setup
 
-All the OpenShift nodes are created in an OpenStack tenant network and as such, can't be accessed directly in most openstack deployments. The installer does not create any floating IP addresses, but does need access to the OpenShift's API as it is being deployed. We will briefly explain how to set up access to the openshift api with and without floating IP addresses.
+If you don't already have the OpenShift Client Tools, follow [these steps](https://docs.openshift.com/enterprise/3.0/cli_reference/get_started_cli.html). Once you have that set up, download the latest stable versions of the [installer](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/). You will need to unzip the tarball with the following command:
 
-### Using Floating IPs
+```sh
+tar -xvf openshift-install-OS-VERSION.tar.gz
+```
 
-This method allows you to attach two floating IP addresses to endpoints in OpenShift.
+For the time being, the OpenStack platform is not supported in the prompts. We will instead provide you with a template `install-config.yaml` to use below. If you want to go beyond the basic customizations we display below, there is more information available in [this document](customization.md). We have labled sections in the yaml below with the following syntax: (#). Look for the numbers below the template for explanations of how to complete those sections.
```

Comment:
Hmm, is that true??

Reply:
It is with the latest 4.2 nightly. We are running a developer preview, and are trying to be as accurate to the current state of the installer.

Reply:
@abhinavdahiya